packages/dds/tree/src/test/feature-libraries/sequence-field/utils.ts

```diff
@@ -34,13 +39,28 @@ export function shallowCompose<T>(changes: TaggedChange<SF.Changeset<T>>[]): SF.
 	});
 }
 
+export function numberTag(integer: number): RevisionTag {
```

Comment:
Do you have a plan to get rid of this eventually?

Reply:
No, I thought this was a fine way to make test code simpler.

Reply:
In that case I think this deserves a comment explaining why this is a reasonable thing to do.

Reply:
Done.